<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QWidget" name="TextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>258</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>44</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <addaction name="action_Open"/>
   <addaction name="action_New"/>
   <addaction name="action_Save"/>
   <addaction name="action_SavaAs"/>
   <addaction name="separator"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Bold"/>
   <addaction name="action_Italic"/>
   <addaction name="action_Underline"/>
   <addaction name="action_Font"/>
   <addaction name="separator"/>
   <addaction name="action_Left"/>
   <addaction name="action_Right"/>
   <addaction name="action_Center"/>
   <addaction name="action_Just"/>
  </widget>
  <widget class="Edit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1001</width>
     <height>651</height>
    </rect>
   </property>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/new.png</normaloff>:/pictures/rsc/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New File</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/open.png</normaloff>:/pictures/rsc/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open the file</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset>
     <normaloff>:/myImages/rsc/close.png</normaloff>:/myImages/rsc/close.png</iconset>
   </property>
   <property name="text">
    <string>Close(&amp;C)</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/save.png</normaloff>:/pictures/rsc/save.png</iconset>
   </property>
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SavaAs">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/saveas.png</normaloff>:/pictures/rsc/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As(&amp;A)</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset>
     <normaloff>:/myImages/rsc/close.png</normaloff>:/myImages/rsc/close.png</iconset>
   </property>
   <property name="text">
    <string>Exit(&amp;X)</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/undo.png</normaloff>:/pictures/rsc/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo(&amp;Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/cut.png</normaloff>:/pictures/rsc/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/copy.png</normaloff>:/pictures/rsc/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/paste.png</normaloff>:/pictures/rsc/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset>
     <normaloff>:/myImages/rsc/find.png</normaloff>:/myImages/rsc/find.png</iconset>
   </property>
   <property name="text">
    <string>Find/Substitue(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset>
     <normaloff>:/myImages/rsc/help.png</normaloff>:/myImages/rsc/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_Bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/bold.png</normaloff>:/pictures/rsc/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold(&amp;B)</string>
   </property>
   <property name="toolTip">
    <string>Bold(B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_Italic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/i.png</normaloff>:/pictures/rsc/i.png</iconset>
   </property>
   <property name="text">
    <string>Italic(&amp;I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Underline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/underline.png</normaloff>:/pictures/rsc/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_Left">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/left.png</normaloff>:/pictures/rsc/left.png</iconset>
   </property>
   <property name="text">
    <string>JustifyLeft</string>
   </property>
  </action>
  <action name="action_Right">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/right.png</normaloff>:/pictures/rsc/right.png</iconset>
   </property>
   <property name="text">
    <string>JustifyRight</string>
   </property>
  </action>
  <action name="action_Center">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/middle.png</normaloff>:/pictures/rsc/middle.png</iconset>
   </property>
   <property name="text">
    <string>JustifyCenter</string>
   </property>
  </action>
  <action name="action_Color">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/color.png</normaloff>:/pictures/rsc/color.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="action_Font">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/size.png</normaloff>:/pictures/rsc/size.png</iconset>
   </property>
   <property name="text">
    <string>Font Setting</string>
   </property>
  </action>
  <action name="action_Just">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/pictures/rsc/just.png</normaloff>:/pictures/rsc/just.png</iconset>
   </property>
   <property name="text">
    <string>Alignment of both ends</string>
   </property>
  </action>
  <action name="action_Y">
   <property name="icon">
    <iconset>
     <normaloff>:/myImages/rsc/redo.png</normaloff>:/myImages/rsc/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo(&amp;Y)</string>
   </property>
   <property name="toolTip">
    <string>Redo(Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_selectAll">
   <property name="icon">
    <iconset>
     <normaloff>:/myImages/rsc/selectall.png</normaloff>:/myImages/rsc/selectall.png</iconset>
   </property>
   <property name="text">
    <string>Select All(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>Select All(A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Edit</class>
   <extends>QTextEdit</extends>
   <header location="global">edit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
